site-packages/integralstor/local_users.py

```diff
@@ -538,6 +538,28 @@ def set_group_membership(grpname, member_users):
         return True, None
 
 
+def get_local_nfs_user_name():
```

Comment:
Should we move these two functions to config.py since it is more of a configurable param rather than a local users specific code?

Reply:
Yeah, done.